scipy/spatial/tests/test__plotutils.py

```diff
@@ -4,8 +4,6 @@
     import matplotlib
     matplotlib.rcParams['backend'] = 'Agg'
     import matplotlib.pyplot as plt
-    from matplotlib.collections import LineCollection
-    from matplotlib import MatplotlibDeprecationWarning
```

Comment:
This is not referenced in the file when it was added in https://github.com/scipy/scipy/pull/5796/files
I ran the tests locally without it and didn't see any errors